examples/python/VideoAPI.ipynb

```diff
@@ -0,0 +1,672 @@
+{
```

Comment:
Can we change the name of the video to follow `snake_case`? None of the files in the repo (apart from from C++ files) use CamelCase

Reply:
do you mean the name of the file?
done if so